e2e_samples/dataset_versioning/insert_sql/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+import sys
+
+sys.path.append('/workspaces/modern-data-warehouse-dataops/e2e_samples/dataset_versioning/insert_sql/')
```

Comment:
We should support running this outside of a Dev Container as well. Can we use a relative path here?

Reply:
When we change to relative path, we need to run pytest only from specific place.

Reply:
I guess we can enforce to run it in pytest directory.